src/esssans/types.py

```diff
@@ -106,6 +106,15 @@ class Filename(sciline.Scope[RunType, str], str):
 """Direct beam after resampling to required wavelength bins"""
 
 
+class DetectorPixelShape(sciline.Scope[RunType, sc.DataGroup], sc.DataGroup):
+    """Geometry of the detector from description in nexus file."""
+
+
+class SampleReferenceTransform(sciline.Scope[RunType, sc.Variable], sc.Variable):
```

Comment:
I think people will find this name confusing, since many instruments may use "reference samples" to compare results to known results from, e.g., other facilities/instruments or.

Maybe `LabFrameTransform`?